php-checks/src/main/java/org/sonar/php/checks/IgnoredReturnValueCheck.java

```diff
@@ -37,7 +37,7 @@
 @Rule(key = "S2201")
 public class IgnoredReturnValueCheck extends PHPVisitorCheck {
 
-  private static final Set<String> PURE_FUNCTIONS = new HashSet<>(Arrays.asList(
+  public static final Set<String> PURE_FUNCTIONS = new HashSet<>(Arrays.asList(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2386)